GeoLib/Polygon.cpp

```diff
@@ -72,7 +72,7 @@ bool Polygon::isPntInPolygon (GeoLib::Point const & pnt) const
 	std::size_t n_intersections (0);
 	GeoLib::Point s;
 
-	if (_simple_polygon_list.empty ()) {
+	if (_simple_polygon_list.front() == this) {
```

Comment:
There are two occasions of `_simple_polygon_list.empty()` left in Polygon.cpp. I guess, they also have to be changed.